test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -68,20 +68,28 @@ public void MatrixFactorizationSimpleTrainAndPredict()
             var data = reader.Read(new MultiFileSource(GetDataPath(TestDatasets.trivialMatrixFactorization.trainFilename)));
 
             // Create a pipeline with a single operator.
-            var options = new MatrixFactorizationTrainer.Options {
+            var options = new MatrixFactorizationTrainer.Options
+            {
                 MatrixColumnIndexColumnName = userColumnName,
                 MatrixRowIndexColumnName = itemColumnName,
                 LabelColumnName = labelColumnName,
                 NumIterations = 3,
                 NumThreads = 1, // To eliminate randomness, # of threads must be 1.
-                K = 7,
+                ApproximationRank = 7,
             };
 
             var pipeline = mlContext.Recommendation().Trainers.MatrixFactorization(options);
 
             // Train a matrix factorization model.
             var model = pipeline.Fit(data);
 
+            // Les's validate content of the model.
+            Assert.Equal(model.Model.ApproximationRank, options.ApproximationRank);
+            var leftMatrix = model.Model.LeftFactorMatrix;
+            var rightMatrix = model.Model.RightFactorMatrix;
+            Assert.Equal(leftMatrix.Count, model.Model.NumberOfRows * model.Model.ApproximationRank);
+            Assert.Equal(rightMatrix.Count, model.Model.NumberOfColumns * model.Model.ApproximationRank);
+
```

Comment:
At least `Assert.Equal()` the first and the last elements in the returned arrays.